pages/feed/index.tsx

```diff
@@ -284,7 +281,7 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
             <FollowingHighlightWrapper selectedFilter={selectedRepo} emojis={emojis} />
           </TabsContent>
         </Tabs>
-        <div className="hidden gap-6 mt-10 md:flex-1 md:flex md:flex-col">
+        <div className="md:hidden gap-6 mt-10 md:flex-1 lg:flex md:flex-col">
```

Comment:
```suggestion
        <div className="md:hidden gap-6 mt-10 w-[30%] lg:w-[24%] flex-none lg:flex md:flex-col">
```

You are also missing a responsive `width` and `flex-none` from the `Right (Filter) Div`. _for the same reason as `Left (userCard) div`_.

...and 

```suggestion
        <div className="hidden gap-6 mt-10 md:flex-1 lg:flex flex-col">
```

Tailwind uses a mobile-first breakpoint system.

This means that unprefixed classes (like `hidden`) take effect on all screen sizes, while prefixed one (like `lg:flex`) only take effect at the specified breakpoint (`lg` in this case) and above.

So `Right Filter Div` will be `hidden` normally on all screen sizes and it will only be in display again at `lg` (desktop) and above i.e. `lg:flex`.

So you needn't specify a responsive class of `md:hidden` since its gonna be hidden normally till `lg`.